### PR TITLE
add 'forgot password?' button to sign in page

### PR DESCRIPTION
**Pull request by ashonuyak**
Changed one more time:

image

### CODE REVIEW

**Comment by oleksandr-danylchenko** [comment]
@ashonuyak 
Could you please leave more space between the "Forgot" and "Sign In" buttons

File: frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx [hunk 1]
```diff
@@ -80,6 +80,13 @@ export const SignInForm = (): React.ReactElement => {
             control={control}
             inputLabel="Password"
           />
+
+          <p className={styles.forgotPassword}>
+            <Link to={AppRoute.FORGOT_PASSWORD} className={styles.link}>
+              Forgot password?
+            </Link>
+          </p>
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx]: Why did you create an additional `p` tag? Couldn't the `Link` be appropriately styled?

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]

File: frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx (hunk 1 above)
Comment (ashonuyak) [reply frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx]: just to reuse styles.link style

Comment (oleksandr-danylchenko) [reply frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx]: We can combine `className`s with the `clsx` utility :slightly_smiling_face: 
What was the issue with the reusage?

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]
@ashonuyak 
Please use symmetric margins:
image

**Review by oleksandr-danylchenko — APPROVED** [review]

**Pull request merged**